### PR TITLE
Create whats-new-in-v72.rst

### PR DESCRIPTION
**Pull request by justinegeffen**

### CODE REVIEW

File: source/welcome/whats-new-in-v72.rst [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+.. note::
+   
+   These boards may not appear in your search results when switching teams.
+
+Workspaces to categories
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you belonged to a workspace at the time of migration you’ll see that they’ve been migrated to custom categories in the sidebar. All boards from a workspace are listed under a category of the same name. Boards from direct messages and group messages appear under the default **Boards** category.
+
+Categories are per-user, and can be renamed or deleted by each user after migration. New users won’t have default categories, and boards they join will appear under the default **Boards** category.
```
Comment (wuwinson) [inline source/welcome/whats-new-in-v72.rst:134]:
```suggestion
Categories are per-user, and can be renamed or deleted by each user after migration. New users won’t see the custom categories created from migration, and boards they join will appear under the default **Boards** category.
```

File: source/welcome/whats-new-in-v72.rst [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+Boards that you create after the migration won’t be linked to a workspace. and will always appear under the default **Boards** category unless you move or hide the boards.
```
Comment (wuwinson) [inline source/welcome/whats-new-in-v72.rst:136]:
```suggestion
Boards that you create after the migration won’t be linked to a workspace, and will always appear under the default **Boards** category unless you move or hide the boards.
```

File: source/welcome/whats-new-in-v72.rst [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+Will I need to backup and restore any data?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The migration will happen automatically, but your administrator should perform a backup prior to the upgrade.
+
+Why are there so many boards on my sidebar?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We removed workspaces, so if you were a member of many boards prior to migration, they will now all appear under the same sidebar. To help you organize your boards better, you can collapse categories, hide boards, and/or move them to other categories.
```
Comment (wuwinson) [inline source/welcome/whats-new-in-v72.rst:149]:
```suggestion
We removed workspaces, so if you were a member of many boards prior to migration, they will now all appear under the same sidebar. To help you organize your boards better, you can collapse categories, hide boards, delete old boards, and/or move them to other categories.
```

File: source/welcome/whats-new-in-v72.rst [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+How do I find my boards now?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the search box (CMD+K/CTRL+K)
```
Comment (wuwinson) [inline source/welcome/whats-new-in-v72.rst:154]:
```suggestion
Use the search box (CMD+K/CTRL+K).
```

File: source/welcome/whats-new-in-v72.rst [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+
+How do I create a new sidebar category?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Select the **…** menu next to the “Boards” (default) category, and select **Create New Category**.
```
Comment (wuwinson) [inline source/welcome/whats-new-in-v72.rst:159]:
```suggestion
Select the **…** menu next to the “Boards” (default) category or any other existing category, and select **Create New Category**.
```

File: source/welcome/whats-new-in-v72.rst [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+
+How do I move a board to another category?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Select the **…** menu next to the board’s name in the sidebar, and select **Move To…**.
+
+How do I remove a board from the sidebar?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Select the **…** menu next to the board’s name, and select **Hide board**.
```
Comment (wuwinson) [inline source/welcome/whats-new-in-v72.rst:169]:
```suggestion
Select the **…** menu next to the board’s name, and select **Hide board**, or if you're an admin of the board, you can delete the board by selecting **Delete board** from the same menu.

**Please note**: Hiding the board only removes the board from _your_ sidebar. You can add the board back to your sidebar using the search box (CMD+K/CTRL+K). Deleting the board permanently removes the board from the sidebar of all board members.
```

File: source/welcome/whats-new-in-v72.rst [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+The boards will be moved to the default “Boards” category. Other users with the same category name will not be impacted.
+
+How do I create a public custom template?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On the template editor, select **Share** and set the team role to **Editor**.
+
+Help, I’ve got a problem not covered here, what do I do?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Please report any issues to the `public Focalboard channel <https://community.mattermost.com/core/channels/focalboard>`_. We’re here to help!
```
Comment (wuwinson) [inline source/welcome/whats-new-in-v72.rst:183]:
```suggestion
Please report any issues to the public `Focalboard channel <https://community.mattermost.com/core/channels/focalboard>`_ or [file a bug](https://github.com/mattermost/focalboard/issues/new?assignees=&labels=Bug%2C+Triage&template=bug_report.md&title=Bug%3A+) on Github. We’re here to help!
```

File: source/welcome/whats-new-in-v72.rst [hunk 8]
```diff
@@ -0,0 +1,183 @@
...
+
+Navigation
+----------
+
+Sidebar categories
+~~~~~~~~~~~~~~~~~~~
+
+Navigation is simplified, and now works similarly to channels. No more switching between workspaces - and the dashboard page has been removed. All the boards you're currently a member of from your current team will appear on the sidebar without needing to switch workspaces. Use the search box (Ctrl+K / Cmd+K) to find additional boards.
+
+.. image:: ../images/boards-whats-new-01.png
```
Comment (wuwinson) [inline source/welcome/whats-new-in-v72.rst:26]: @justinegeffen Can we reduce the width of this image to 303px so it's the same width as the 2nd image after this one?

File: source/welcome/whats-new-in-v72.rst [hunk 9]
```diff
@@ -0,0 +1,183 @@
...
+  :alt: You can create a board from the RHS which is automatically added to the channel you're in.
+
+Sharing
+-------
+
+Select **Share** to view and edit permissions to a board.
+
+.. image:: ../images/boards-whats-new-06.png
+  :alt: Select the share button to open up the permissions options.
+
```
Comment (wuwinson) [inline source/welcome/whats-new-in-v72.rst:68]: Can we force a line break between these 2 images?

**Review by wuwinson — CHANGES_REQUESTED** [review]
@justinegeffen Added a few minor changes. LMK what you think!

File: source/welcome/whats-new-in-v72.rst (hunk 8 above)
Comment (justinegeffen) [reply source/welcome/whats-new-in-v72.rst:26]: Sure. Let me know if that looks better.

File: source/welcome/whats-new-in-v72.rst (hunk 9 above)
Comment (justinegeffen) [reply source/welcome/whats-new-in-v72.rst:71]: Let me know if it looks better?

**Review by wuwinson — APPROVED** [review]
LGTM!

**Pull request merged**